util-bukkit/src/main/i18n/templates/strings.properties

```diff
@@ -908,134 +907,123 @@ misc.appeals.singular = appeal
 misc.appeals.plural = appeals
 misc.on = On
 misc.off = Off
-noPermissions = You do not have permission.
-commands.noPermission = You don't have permission.
-console = Console
+misc.noPermission = You don't have permission.
+misc.console = Console
 
 
 # {0} = "1"
-time.interval.millisecond = {0} millisecond
+misc.time.millisecond = {0} millisecond
 # {0} = an amount of milliseconds (a number)
-time.interval.milliseconds = {0} milliseconds
+misc.time.milliseconds = {0} milliseconds
 # {0} = "1"
-time.interval.second = {0} second
+misc.time.second = {0} second
 # {0} = an amount of seconds (a number)
-time.interval.seconds = {0} seconds
+misc.time.seconds = {0} seconds
 # {0} = "1"
-time.interval.minute = {0} minute
+misc.time.minute = {0} minute
 # {0} = an amount of minutes (a number)
-time.interval.minutes = {0} minutes
+misc.time.minutes = {0} minutes
 # {0} = "1"
-time.interval.hour = {0} hour
+misc.time.hour = {0} hour
 # {0} = an amount of hours (a number)
-time.interval.hours = {0} hours
+misc.time.hours = {0} hours
 # {0} = "1"
-time.interval.day = {0} day
+misc.time.day = {0} day
 # {0} = an amount of days (a number)
-time.interval.days = {0} days
-
-# {0} = an amount of time in the form of [NUMBER + ms/s/m/h/d]
-# Examples: 32ms, 1d, 8h, 43m
-time.ago = {0} ago
+misc.time.days = {0} days
+# {0} = an amount of time
+misc.time.ago = {0} ago
 
-prefixed.tip = Tip
-prefixed.alert = Alert
+misc.tip = Tip
+misc.alert = Alert
+misc.warning = WARNING
 
 # {0} = a player name
-tip.teleportTo = Teleport to {0}
+misc.teleportTo = Teleport to {0}
 
 +           -----OTHER GAMEMODES-----
 + Strings belonging to smaller gamemodes that don't have enough text for
 + their own category
 
-map.gamemode.short.tdm = TDM
-map.gamemode.short.ad = A/D
-map.gamemode.short.koth = KoTH
-map.gamemode.short.blitz = Blitz
-map.gamemode.short.rage = Rage
-map.gamemode.short.scorebox = Scorebox
-map.gamemode.short.arcade = Arcade
-map.gamemode.short.ffa = FFA
-map.gamemode.short.mixed = Mixed
-
-map.gamemode.long.tdm = Team Deathmatch
-
-
-map.gamemode.long.ad = Attack/Defend
-map.gamemode.long.koth = King of the Hill
-map.gamemode.long.blitz = Blitz
-map.gamemode.long.rage = Rage
-map.gamemode.long.scorebox = Scorebox
-map.gamemode.long.arcade = Arcade
-map.gamemode.long.ffa = Free-for-all
-map.gamemode.long.mixed = Mixed
+gamemode.tdm.name.short = TDM
```

Comment:
On second thought, how about we move all the game modes titles into this section? So they are in 1 place.

Reply:
Very good point, gamemodes are not necessarily connected to any objective type.